lib/omise/OmiseCapabilities.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+class OmiseCapabilities extends OmiseApiResource
+{
+    const ENDPOINT = 'capability';
+
+    /**
+     * @var array  of the filterable keys.
+     */
+    const FILTERS = array(
+        'backend' => array('currency', 'type', 'chargeAmount')
+    );
+
+    protected function __construct($publickey = null, $secretkey = null)
+    {
+        parent::__construct($publickey, $secretkey);
+        $this->setupFilterShortcuts();
```

Comment:
Again, I have a really big problem with not allowing underscore prefixes to name `protected` and `private` members. Without these, the code is less readable. I will continue to use them

Reply:
@jonrandy https://www.php-fig.org/psr/psr-2/#43-methods

Reply:
I know, but I have a really big problem with it and I don't think we should do it

Reply:
Again, this topic depends on personal taste. So I picked the one without `_` to make code consistency with the rest. And also to comply with PSR-2 & PHP Code Sniffer standard.

Reply:
Code Sniffer can use any set of rules. We could inherit from PSR-2 and switch off rules we don't like